dbus/machine-learning-agent.conf.in

```diff
@@ -5,13 +5,13 @@
         <allow send_destination="org.tizen.machinelearning.service"
 		send_interface="org.tizen.machinelearning.service.pipeline"/>
     </policy>
-    <policy user="system_fw">
+    <policy user="service_fw">
         <allow own="org.tizen.machinelearning.service"/>
     </policy>
     <policy context="default">
-        <allow own="org.tizen.machinelearning.service"/>
-        <allow send_destination="org.tizen.machinelearning.service" send_type="method_call"/>
-        <allow own="org.tizen.machinelearning.service"/>
-        <allow send_destination="org.tizen.machinelearning.service.pipeline" send_type="method_call"/>
+	<deny own="org.tizen.machinelearning.service"/>
+	<deny send_destination="org.tizen.machinelearning.service"/>
+
+	<allow send_destination="org.tizen.machinelearning.service"/>
```

Comment:
contradicting LINE 13 and LINE 15??
(if you confirm that this is intentional and not a mistake, please merge.)

Reply:
As I mentioned above, this is an intended workaround. After creating Tizen privilege, this policy will be updated.